website/docs/r/branch_policy_auto_reviewers.html.markdown

```diff
@@ -0,0 +1,90 @@
+---
+layout: "azuredevops"
+page_title: "AzureDevops: azuredevops_branch_policy_auto_reviewers"
+description: |-
+  Manages required reviewer policy branch policy within Azure DevOps project.
+---
+
+# azuredevops_branch_policy_auto_reviewers
+
+Manages required reviewer policy branch policy within Azure DevOps.
+
+## Example Usage
+
+```hcl
+resource "azuredevops_project" "p" {
+  project_name = "Sample Project"
+}
+
+resource "azuredevops_git_repository" "r" {
+  project_id = azuredevops_project.p.id
+  name       = "Sample Repo"
+  initialization {
+    init_type = "Clean"
+  }
+}
+
```

Comment:
Can you format this terraform? Put it in a tf file, format it and paste it back? I can see a few things not being spaces correctly

Reply:
better now? terraform extension on vscode driving me crazy 😶

Reply:
Looks great! I'm sorry, I'm channeling @nmiodice with this level of nitpicking. 😆 (He's told me off so many times for my improper terraform in docs) 😄